.github/workflows/add_pr2project.yml

```diff
@@ -1,155 +0,0 @@
-# This workflow adds a PR in this repository to the main Neurobagel project (#1)
-# when and if it is labeled with a label of interest.
-# Currently our labels of interest are
-# - _bot
-# - _community
-# 
-# Once added to the project, we also set the custom field "Status" to "Community"
-# for the PR. This in effect adds the PR to our main board in the "Community" column.
-
-name: PR automation
-on:
-  pull_request:
-    types: [ labeled ]
-
-jobs:
-  add2project:
-    runs-on: ubuntu-latest
-    steps:
-      - name: Check if label of interest is applied
-        id: preview_label_check
-        uses: docker://agilepathway/pull-request-label-checker:latest
-        with:
-          any_of: _bot,_community
-          repo_token: ${{ secrets.NB_PROJECT_PAT }}
-          allow_failure: true
-      
-      - name: Detect if PR is in project
-        if: steps.preview_label_check.outputs.label_check == 'success'
-        id: in_project
-        env:
-          GH_TOKEN: ${{ secrets.NB_PROJECT_PAT }}
-        run: | 
-          gh pr view ${{ github.event.pull_request.html_url }} --json projectItems | jq -r '.projectItems[].title' | grep Neurobagel
-        continue-on-error: true
-
-      - name: Add PR to Project
-        if: steps.in_project.outcome == 'failure' && steps.preview_label_check.outputs.label_check == 'success' 
-        uses: actions/add-to-project@main
-        with:
-          project-url: https://github.com/orgs/neurobagel/projects/1
-          github-token: ${{ secrets.NB_PROJECT_PAT }}
-
-        # Note that in contrast to the graphical github UI,
```

Comment:
I know you refactored the workflow but I still found these comments very useful for understanding what's going on in the `Find project node id` and `Get IDs for Status field and Community option` steps of the `process_prs` job in the new yml below.

Could we add these comments back into the new workflow?